src/BlockTypes/ProductQuery.php

```diff
@@ -325,6 +326,34 @@ function ( $carry, $item ) {
 	 * @return array
 	 */
 	private function get_stock_status_query( $stock_statii ) {
+		$stock_status_options = array_keys( wc_get_product_stock_status_options() );
+
+		if ( ! is_array( $stock_statii ) ) {
+			return array();
+		}
```

Comment:
Not a big deal, but should we return even earlier? We don't even need a call to `wc_get_product_stock_status_option` if the argument is not an array.

Reply:
Absolutely!